cluster/gce/gci/configure-helper.sh

```diff
@@ -3315,16 +3315,32 @@ oom_score = -999
   default_runtime_name = "runc"
 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
   runtime_type = "io.containerd.runc.v2"
-[plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
```

Comment:
this format is deprecated and nothing should be pulling through k8s.gcr.io anymore anyhow

ditto dockerhub, our CI shouldn't be pulling from it, and GCR is deprecated